src/lib/utils/formatLocaleNumber.ts

```diff
@@ -0,0 +1,17 @@
+import { Currency, CurrencyAmount, Price } from '@uniswap/sdk-core'
+import { SupportedLocale } from 'constants/locales'
+
+interface FormatLocaleNumberArgs {
+  number: CurrencyAmount<Currency> | Price<Currency, Currency> | number
+  locale: SupportedLocale | null | undefined
+  options?: Intl.NumberFormatOptions
+  sigFigs?: number
+}
+
+export default function formatLocaleNumber({ number, locale, sigFigs, options = {} }: FormatLocaleNumberArgs): string {
+  if (typeof number === 'number') {
+    return number.toLocaleString(locale ? [locale, 'en-US'] : 'en-US', options)
```

Comment:
use DEFAULT_LOCALE

Reply:
Also, pull out this defaulting: `locale = locale ? [locale, DEFAULT_LOCALE] : DEFAULT_LOCALE`